universal-application-tool-0.0.1/app/assets/javascripts/main.ts

```diff
@@ -159,11 +159,35 @@ function attachLineClampListeners() {
   applicationCardDescriptions.forEach(el => el.addEventListener("click", removeLineClamp));
 }
 

```

Comment:
Can you put the modal specific scripts in its own class? Now that we're piling on the JS we should move away from using main.ts.

```
class ModalController {
    attachListeners() {
       ...
    }
    
    dismissModal(modal) {
       ...
    }
}
```

Reply:
got it!